rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/restdocs/databaseSanitationSpecTest.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Siemens AG, 2023-2024.
+ * Part of the SW360 Portal Project.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.eclipse.sw360.rest.resourceserver.restdocs;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.apache.thrift.TException;
+import org.eclipse.sw360.datahandler.thrift.components.ComponentService;
+import org.eclipse.sw360.datahandler.thrift.projects.ProjectService;
+import org.eclipse.sw360.datahandler.thrift.users.User;
+import org.eclipse.sw360.rest.resourceserver.TestHelper;
+import org.eclipse.sw360.rest.resourceserver.DatabaseSanitation.Sw360DatabaseSanitationService;
+import org.eclipse.sw360.rest.resourceserver.user.Sw360UserService;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.hateoas.MediaTypes;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+public class databaseSanitationSpecTest extends TestRestDocsSpecBase {
+    @Value("${sw360.test-user-id}")
+    private String testUserId;
+
+    @Value("${sw360.test-user-password}")
+    private String testUserPassword;
+    
+    @MockBean
+    private Sw360UserService userServiceMock;
+    
+    @MockBean
+    private Sw360DatabaseSanitationService sanitationService;
+    
+    @MockBean
+    ComponentService.Iface componentClient;
+    
+    @MockBean
+    ProjectService.Iface projectClient;
+private HttpServletRequest request;
```

Comment:
this variable `request` is never used. Please remove this.

Reply:
done.